src/core/Core_result.ml

```diff
@@ -100,7 +100,7 @@ type t = {
   (* old: matches : Pattern_match.t list *)
   processed_matches : processed_match list;
   errors : Core_error.t list;
-  scanned : Fpath.t list;
+  scanned : Target.t list;
```

Comment:
Maybe add a comment like
(* old: was simply [Fpath.t list] before, but for inter-file diff scan metrics we also need
 * to know the engine_kind used on the target, hence the use of [Target.t list] now
 * )
 *

Reply:
done.